cfn-resources/resource-policy/example_inputs/inputs_1_create.json

```diff
@@ -0,0 +1,12 @@
+{
+  "TPSCode": "...",
```

Comment:
what is this?

Reply:
removed example_inputs, they were unnecessary files